apps/examples/webserver/webserver_main.c

```diff
@@ -111,25 +111,27 @@ static int http_send_chunk(struct http_client_t *client)
 	memset(msg_body, 'a', MAX_DATA_SIZE);
 	msg_body[MAX_DATA_SIZE] = 0;
 
-	ret = http_send_response_chunk(client, 200, "OK", msg_body, NULL, FIRST_DATA);
+	ret = http_send_response_chunk(client, 200, "OK", msg_body, MAX_DATA_SIZE, NULL, FIRST_DATA);
```

Comment:
Similar to @Taejun-Kwon 's comment on line 121,
Better to use Macro definitions for custom integers unless they are standard values like 0666/0777 for file permissions.

Reply:
done